client/ayon_houdini/startup/otls/AyonPub.1.1.hda/Driver_1AyonPub_8_81.1/DialogScript

```diff
@@ -0,0 +1,690 @@
+# Dialog script for AyonPub::1.1 automatically generated
+
+{
+    name	AyonPub::1.1
+    script	AyonPub
+    label	AyonPub
+
+    help {
+	""
+    }
+
+    inputlabel	1	"Input #1"
+    inputlabel	2	"Input #2"
+    inputlabel	3	"Input #3"
+    inputlabel	4	"Input #4"
+    inputlabel	5	"Input #5"
+    inputlabel	6	"Input #6"
+    inputlabel	7	"Input #7"
```

Comment:
Oh lord. Can we clean this up? Or do we need this for all the inputs? :') (Seems odd?)

Reply:
i did it the way that the deadline node dose it because if you dont set it to 9999 inputs you "limit" the amount of nodes that can be connected to the connection block. 
the block atop of a node is actually just the inputs in a group representation. 

that's how i understand it.

Reply:
![image](https://github.com/user-attachments/assets/5d2292df-b2e7-4285-bc2b-571b841c6647)
this happens when you do 1 input. 
so if we want to allow the connection of multiple upstream nodes like the other rpos do we will need them like this. 

i guess we could reduce the amount but tbh i dont think this is gonna win us anything. 

just to mention it i believe multiple upstream nodes connected like a merge node is a nice convenience feature for users otherwise they would probably just plug a merge node into our node.

Reply:
Stupid Houdini I suppose - why can't we just set num inputs to 1000 but skip the labels. Well, fine with me 🤷